src/main/js/helpers/constants.js

```diff
@@ -213,7 +219,7 @@ export default {
     DEFAULT_REGION_LINE_STROKE_WIDTH: 5,
     DEFAULT_BAR_SELECTION_PADDING: 5,
     DEFAULT_ACTIVITY_BAR_HASH_COLOR: "#d3d4d5",
```

Comment:
Does this color follow the 3:1 contrast ratio? if it does not follow, Should we update this color too the 3:1 ratio?

Reply:
We do have 3 other colors in constants which do not meet 3:1 contrast ratio. We need to discuss with ux team about the change of color.